CHANGES

```diff
@@ -1,4 +1,30 @@
 
+Changes with Unit 1.32.0                                         26 Feb 2024
```

Comment:
Placeholder comment: the 26 Feb date may change by a few days. Need to ensure it doesn't conflict with any dates on the internal [[release calendar]](https://nginxsoftware.atlassian.net/wiki/spaces/ENGUSEMEA/pages/446333086). 27th and 29th look free right now; will coordinate with release management folks to lock in a date.

Reply:
Let's aim for 27 Feb 2024 here and in the `changes.xml`